src/main/java/com/google/sps/servlets/FindLocationServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/find-location")
+public class FindLocationServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    response.getWriter().println("<h1>Find Location</h1>");
+  }
+
+  @Override
```

Comment:
Why include these doPost functions? If the intention is to only use get requests, then only define get requests

Reply:
so if i understand correctly, we are making get requests to these endpoints with the correlator as a parameter. is this a good way/style to send stuff to the server without using post bc a get request is not meant for that purpose?

Reply:
IMO I think its ok to use get requests for these even though you aren't really "getting" any information, since its pretty standard to use get requests for tracking pings in VAST (often servers will respond with a 1x1 pixel image, or just an empty 200 response).  If you use the simid spec's ReportTracking function, it will trigger a get request.  

I think its fine to just use get requests for this then, and you can just remove the doPost definitions